samples/queuesGettingStarted.ts

```diff
@@ -0,0 +1,78 @@
+import { OnMessage, OnError, MessagingError, delay, ServiceBusMessage, ReceiveMode, generateUuid, Namespace } from "../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+async function main(): Promise<void> {
+  await sendMessage();
+  await receiveMessage();
+}
+
+async function sendMessage(): Promise<void> {
+  let nsSend: Namespace;
+  nsSend = Namespace.createFromConnectionString(str);
+  const sendClient = nsSend.createQueueClient(path);
+  var data = [
+    { name: "Einstein", firstName: "Albert" },
+    { name: "Heisenberg", firstName: "Werner" },
+    { name: "Curie", firstName: "Marie" },
+    { name: "Hawking", firstName: "Steven" },
+    { name: "Newton", firstName: "Isaac" },
+    { name: "Bohr", firstName: "Niels" },
+    { name: "Faraday", firstName: "Michael" },
+    { name: "Galilei", firstName: "Galileo" },
+    { name: "Kepler", firstName: "Johannes" },
+    { name: "Kopernikus", firstName: "Nikolaus" }
+  ];
+  try {
+    for (let i = 0; i < data.length; i++) {
+      var message = {
+        body: JSON.stringify(data[i]),
+        contentType: "application/json",
+        label: "Scientist",
+        messageId: generateUuid(),
+      }
+      await sendClient.send(message);
+      console.log("Sent message number:", i + 1);
+    }
+    console.log("\n>>>>>> Total Sent messages: %d\n", data.length);
+  } catch (err) {
+    console.log("Sending error", err);
```

Comment:
For a second I thought we were "sending" the error :)
Change this to "Error while sending"?

Reply:
Ok!